v2/thirdparty/common-customizations/getting-provider-access-token.mdx

```diff
@@ -4,179 +4,6 @@ title: Getting provider's access token
-							resp, err := originalSignInUpPOST(provider, code, authCodeResponse, redirectURI, options, userContext)
-							if err != nil {
-								return tpmodels.SignInUpPOSTResponse{}, err
-							}
-
-							if resp.OK != nil {
-								// the user logged in / signed up successfully
-
-								// In this example we are using Google as our provider
-								authCodeResponse := resp.OK.AuthCodeResponse
-
-								accessToken := authCodeResponse.(map[string]interface{})["access_token"].(string)
-
-								fmt.Println(accessToken)
-
-							}
-
-							return resp, err
-						}
-
-						return originalImplementation
-					},
-				},
-			}),
-		},
-	})
-}
-```
-
-</TabItem>
-<TabItem value="python">
-
-```python
-from supertokens_python import init, InputAppInfo
-from supertokens_python.recipe import thirdparty
-from supertokens_python.recipe.thirdparty.interfaces import APIInterface, APIOptions, SignInUpPostOkResult
-from typing import Union, Dict, Any
-from supertokens_python.recipe.thirdparty.provider import Provider
-
-# highlight-start
-def override_thirdparty_apis(original_implementation: APIInterface):
-    
-    original_sign_in_up_post = original_implementation.sign_in_up_post
-    
-    async def sign_in_up_post(provider: Provider, code: str, redirect_uri: str, client_id: Union[str, None], auth_code_response: Union[Dict[str, Any], None], api_options: APIOptions,
-                              user_context: Dict[str, Any]):
-        # First we call the original implementation of signInPOST.
-        response = await original_sign_in_up_post(provider, code, redirect_uri, client_id, auth_code_response, api_options, user_context)
-        
-        # Post sign up response, we check if it was successful
-        if isinstance(response, SignInUpPostOkResult):
-            _ = response.user.user_id
-            __ = response.user.email
-
-            # In this example we are using Google as our provider
-            thirdparty_auth_response = response.auth_code_response
-
-            if thirdparty_auth_response is None:
-                raise Exception("Should never come here")
-
-            access_token = thirdparty_auth_response["access_token"]
-            print(access_token)
-            # TODO
-
-        return response
-        
-    original_implementation.sign_in_up_post = sign_in_up_post
-    return original_implementation
-# highlight-end
-
-init(
-    app_info=InputAppInfo(api_domain="...", app_name="...", website_domain="..."),
-    framework='...', # type: ignore
-    recipe_list=[
-        thirdparty.init(
-            # highlight-start
-            override=thirdparty.InputOverrideConfig(
-                apis=override_thirdparty_apis
-            ),
-            # highlight-end
-            sign_in_and_up_feature=thirdparty.SignInAndUpFeature(providers=[
-                #...
-            ])
-        )
-    ]
-)
-```
-
-</TabItem>
-</BackendSDKTabs>
+[Need to create page for redirect](./TODO)
```

Comment:
the page already exists? It's the getting user info page right? or somewhere else?